wagmi.config.ts

```diff
@@ -1,2029 +1,1388 @@
+    outputs: [{ internalType: 'bool', name: '', type: 'bool' }],
+    stateMutability: 'nonpayable',
+    type: 'function',
   },
   {
-      "inputs": [
-          {
-              "internalType": "uint256",
-              "name": "length",
-              "type": "uint256"
-          }
-      ],
-      "name": "setActivePeriodLength",
-      "outputs": [],
-      "stateMutability": "nonpayable",
-      "type": "function"
-  },
-  {
-      "inputs": [
-          {
-              "internalType": "uint256",
-              "name": "amount",
-              "type": "uint256"
-          }
-      ],
-      "name": "setPeriodDistributionAmount",
-      "outputs": [],
-      "stateMutability": "nonpayable",
-      "type": "function"
-  }
+    inputs: [{ internalType: 'address', name: 'newOwner', type: 'address' }],
+    name: 'transferOwnership',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+  {
+    inputs: [],
+    name: 'version',
+    outputs: [{ internalType: 'string', name: '', type: 'string' }],
+    stateMutability: 'view',
+    type: 'function',
+  },
 ] as const
 
+const mainnetRiverTokenAddress = '0x53319181e003E7f86fB79f794649a2aB680Db244' satisfies Address
+const baseRiverTokenAddress = '0x91930fd11ABAa5241241d3B07c02A8d0B5ac1920' satisfies Address
+const baseGammaSepoliaTokenAddress = '0x24e3123E1b30E041E2df26Da9d6140c5B07Fe4F0' satisfies Address
+
 export default defineConfig({
   out: 'src/contracts.ts',
   contracts: [
     {
-      abi: BaseRiverTokenAbi,
-      name: 'BaseRiverToken',
+      name: 'RiverToken',
       address: {
-        [base.id]: '0x9172852305F32819469bf38A3772f29361d7b768',
-        [baseSepolia.id]: '0x49442708a16Bf7917764F14A2D103f40Eb27BdD8',
+        [mainnet.id]: mainnetRiverTokenAddress,
+        [base.id]: baseRiverTokenAddress,
+        [baseSepolia.id]: baseGammaSepoliaTokenAddress,
       },
+      abi: riverTokenAbi,
     },
     {
-      abi: RiverClaimerAbi,
-      name: 'RiverClaimer',
+      name: 'RewardsDistribution',
       address: {
-        [base.id]: '0x7c0422b31401C936172C897802CF0373B35B7698',
-        [baseSepolia.id]: '0x08cC41b782F27d62995056a4EF2fCBAe0d3c266F',
+        [base.id]: omega.address as Address,
+        [baseSepolia.id]: gamma.address as Address,
       },
-    }
+      abi: rewardDistributionAbi,
+    },
   ],
   plugins: [
     etherscan({
-      apiKey: env.ETHERSCAN_API_KEY!,
+      apiKey: env.ETHERSCAN_API_KEY,
       chainId: mainnet.id,
       contracts: [
         {
-          name: 'RiverToken',
-          address: {
-            [mainnet.id]: '0x53319181e003e7f86fb79f794649a2ab680db244',
-            [sepolia.id]: '0x40ef1bb984503bb5adef041a88a4f9180e8586f9',
-          },
-        },
-        {
-          name: 'RiverAuthorizer',
+          name: 'Authorizer',
           address: {
             [mainnet.id]: '0x0bEe55b52d01C4D5d4D0cfcE1d6e0baE6722db05',
             [sepolia.id]: '0x2f5E8F6Fb7EcF63d13C13B698d1e0B3EA4Ef604B',
```

Comment:
unsure if that's the most up-to-date address of the authorizer contract - if it is, we should document it in the river docs - only found one reference matching this mainnet address in the river repo (hardcoded in a solidity file - not even inside /contracts or /packages/generated)